.storybook/4.SHADOW.stories.mdx

```diff
@@ -144,8 +144,8 @@ As well as the neutral colors for shadow 2 other colors exist that are used for
 | Color       | CSS                           |
 | ----------- | ----------------------------- |
 | **neutral** | `var(--color-shadow-default)` |
-| **primary** | `var(--color-primary-shadow)` |
-| **danger**  | `var(--color-error-shadow)`   |
+| **primary** | `var(--color-shadow-primary)` |
+| **danger**  | `var(--color-shadow-error)`   |
```

Comment:
In [design tokens v4](https://github.com/MetaMask/design-tokens/blob/main/MIGRATION.md#css-1) `--color-primary-shadow` and `--color-error-shadow` where changed to `--color-shadow-primary` and `--color-shadow-error`. All instances have been replaced and primary and error shadows still work

<img width="727" alt="Screenshot 2024-06-09 at 8 57 43 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0b498be3-3990-4de2-9166-c46e83a0c0dc">

![Screenshot 2024-06-09 at 8 59 20 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/a8485c36-c3bf-4262-894b-db6714192349)
![Screenshot 2024-06-09 at 9 00 32 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/0ebbc999-bb3f-4a87-a7ee-76e7148c9f17)

Screenshot shows primary shadow still working after v4 upgrade

<img width="1510" alt="Screenshot 2024-06-09 at 9 06 17 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/7d87fce6-30c8-44bd-b360-08cf6dbc46d5">
<img width="755" alt="Screenshot 2024-06-09 at 9 07 06 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/e3ae767e-d6d3-46e5-8d03-5de37215f2f3">